frontend/src/Routing.js

```diff
@@ -7,19 +7,21 @@ import TableListOfWorkSheets from './worksheets/TableListOfWorkSheets'
 import TableListofPartners from './partners/TableListofPartners'
 import CreateWorkSheet from './worksheets/CreateWorkSheet'
 import LoadingScreen from './LoadingScreen'
+import useCurrentUser from './hooks/useCurrentUser'
 
 function Routing() {
+  const { isAdmin } = useCurrentUser()
   return (
     <div className="col-12">
       <Switch>
         <Route path="/employees/new">
-          <CreateEmployee />
+          {isAdmin ? <CreateEmployee /> : <Redirect to="/" />}
```

Comment:
Lehetne sokkal egyszerűbben, ha az összes admin Route-ot összefognád egy Fragment-be és az raknád feltételesen be. Még a redirect sem kell ilyen esetben.